arch/HPPA/HPPADisassembler.c

```diff
@@ -19,6 +19,10 @@
 #define GET_BIT(X, WHICH) \
   GET_FIELD (X, WHICH, WHICH)
 
+#define CMPLT_IS_REG_MOD(CMPLT) (((CMPLT) & 1) == 1)
```

Comment:
Sorry, I was thinking about something like
```c
if (cmplt & HPPA_REG_MOD_FLAG)
...
```
Having something like `HPPA_CMPLT_REG_MOD` in an enum, is better than my first request for a macro.

```
typedef enum {
  HPPA_CMPLT_REG_MOD = 1;
  HPPA_CMPLT_SOME_OTHER_FLAG = ...

} hppa_cmplt_flags
```

But if `cmplt` will only ever have a single bit set and contains no other flags, this is ok.

Reply:
I only need to check if completer of load/store instruction has "modify base register" bit. If it is true, than I add write access to base register